.travis.yml

```diff
@@ -63,28 +65,12 @@ branches:
 git:
   depth: 1
 
-# Clones WordPress and configures our testing environment
 before_script:
-    - phpenv config-rm xdebug.ini
-    - export PLUGIN_SLUG=$(basename $(pwd))
-    - source ~/.nvm/nvm.sh
-    - export PATH="$HOME/.composer/vendor/bin:$PATH"
-    - |
-        if [[ ${TRAVIS_PHP_VERSION:0:2} == "7." ]]; then
-            composer global require "phpunit/phpunit=5.7.*"
-        elif [[ ${TRAVIS_PHP_VERSION:0:3} != "5.2" ]]; then
-            composer global require "phpunit/phpunit=4.8.*"
-        fi
-    - nvm install 8
-    - nvm use 8
-    - ./tests/prepare-wordpress.sh
-    - mysql -e "set global wait_timeout = 3600;"
+  - export PLUGIN_SLUG=$(basename $(pwd))
+  - export PATH="$HOME/.composer/vendor/bin:$PATH"
```

Comment:
We want to inherit these constants to `run-travis.sh`, thus they should be here and not in `setup-travis.sh`